lib/Sema/CSFix.h

```diff
@@ -1633,6 +1639,22 @@ class SpecifyClosureReturnType final : public ConstraintFix {
                                           ConstraintLocator *locator);
 };
 
+class SpecifyObjectLiteralTypeImport final : public ConstraintFix {
+  SpecifyObjectLiteralTypeImport(ConstraintSystem &cs, ConstraintLocator *locator)
+      : ConstraintFix(cs, FixKind::SpecifyObjectLiteralTypeImport, locator) {}
+
+public:
+  std::string getName() const {
+    return "specify object literal protocol type import";
```

Comment:
Nit: would be better if it mentioned something like `import required module to gain access to a default literal type`.